packages/eval/harbor/egress_filter.py

```
@@ -103,14 +103,52 @@ def public_trajectory_repository(host: str, path_query: str) -> bool:
 
 
+def response(flow: object) -> None:
+    response = getattr(flow, "response", None)
+    if getattr(response, "status_code", None) != 101:
+        return
+    headers = getattr(response, "headers", None)
+    upgrade = ""
+    if headers is not None:
+        try:
+            upgrade = str(headers.get("upgrade", "") or "")
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)